tests/unit/matrix/test_measure.py

```diff
@@ -1290,4 +1337,49 @@ def it_computes_its_blocks_to_help(self, request, dimensions_):
 
     @pytest.fixture
     def dimensions_(self, request):
-        return (instance_mock(request, Dimension), instance_mock(request, Dimension))
+        return instance_mock(request, Dimension), instance_mock(request, Dimension)
+
+
+class Describe_Zscores(object):
+    """Unit test suite for `cr.cube.matrix.measure._Zscores` object."""
+
+    def it_computes_zscore_subtotals_blocks_to_help(self, request, dimensions_):
```

Comment:
`ZscoreSubtotals.blocks` is an interface method (not a helper) and so the `_to_help` suffix is not appropriate for this test (or the next one).